internal/keylight/temperature_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Terin Stock
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keylight_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/mdlayher/flightdeck/internal/keylight"
+)
+
+func TestMarshalTemp(t *testing.T) {
```

Comment:
How about combining both tests so we do a round trip conversion for each case. You can add the cases from below to this function's table.